Command/GenerateCommand.php

```diff
@@ -43,6 +43,7 @@ protected function configure()
 
         $this->addArgument('bundle', InputArgument::IS_ARRAY, 'The bundle name to "easy-extends"');
         $this->addOption('dest', 'd', InputOption::VALUE_OPTIONAL, 'The base folder where the Application will be created', false);
+        $this->addOption('namespace', 'm', InputOption::VALUE_OPTIONAL, 'The namespace for the classes', false);
```

Comment:
And `n` would be a better choice IMHO.

After that, the option short name is not mandatory.